actionpack/test/controller/support/custom_assertions.rb

```diff
@@ -0,0 +1,3 @@
+def assert_filtered_out(params, key)
+  assert !params.has_key?(key), "key #{key.inspect} has not been filtered out"
+end
```

Comment:
Can we find a solution that does not add global methods? I'd rather have these tests include a module than adding a method that's automatically available as soon as the file gets required.

Also, if we do that, `custom_assertions` seems too generic I think? It could be a bit more specific to what it contains.